src/resources/lib/pdfjs/web/viewer.js

```diff
@@ -4174,11 +4228,13 @@ class EventBus {
   dispatch(eventName) {
     const eventListeners = this._listeners[eventName];
 
+    const args = Array.prototype.slice.call(arguments, 1);
+    console.log("PDF EVENT BUS", eventName, args);
+
```

Comment:
Is it okay to remove this change? (this is just debug info, right?)

Reply:
Reference:

https://github.com/edrlab/pdf.js/blob/b04b7e5bf8ae5efe8a6e0c6d98e042f89ad928d4/web/ui_utils.js#L845-L846